Stellar-operation-manage-limits.x

```diff
@@ -59,39 +75,49 @@ struct ManageLimitsOp
 
 /******* ManageLimits Result ********/
 
+//: Result codes for ManageLimits operation
 enum ManageLimitsResultCode
 {
     // codes considered as "success" for the operation
+    //: `ManageLimitsOp` was successfully performed
     SUCCESS = 0,
 
     // codes considered as "failure" for the operation
+    //: (reserved for future use) Invalid input
     MALFORMED = -1,
+    //: Limits entry not found
     NOT_FOUND = -2,
+    //: (reserved for future use) Limits entry already exists
     ALREADY_EXISTS = -3,
-    CANNOT_CREATE_FOR_ACC_ID_AND_ACC_TYPE = -4, // limits cannot be created for account ID and account type simultaneously
+    //: Limits cannot be created for account ID and account role simultaneously
+    CANNOT_CREATE_FOR_ACC_ID_AND_ACC_TYPE = -4, // FIXME ACC_ROLE ?
+    //: Limits entry is invalid (e.g. weeklyOut is less than dailyOut)
     INVALID_LIMITS = -5
 };
 
+//: `ManageLimitsResult` defines result of ManageLimits operation based on the given `ManageLimitsResultCode`
 union ManageLimitsResult switch (ManageLimitsResultCode code)
 {
 case SUCCESS:
     struct {
+        //: `details` represents the details of the ManageLimits operation result
```

Comment:
too much "details". 
... is used for additional information regarding action performed...